src/platform/forms-system/src/js/web-component-patterns/fileInputPattern.jsx

```diff
@@ -61,47 +62,62 @@ export const filePresenceValidation = (
  *  formHeading?: UIOptions['formHeading'],
  *  formDescription?: UIOptions['formDescription'],
  *  formHeadingLevel?: UIOptions['formHeadingLevel'],
- * }} stringOrOptions
+ * }} options
```

Comment:
Removed the option to just pass a string as we require the developer to explicitly pass a `required` prop, like for checkbox group